modules/installation-configuration-parameters.adoc

```diff
@@ -387,6 +387,10 @@ endif::bare[]
 The IP address blocks for machines.
 
 If you specify multiple IP address blocks, the blocks must not overlap.
+
+ifdef::ibm-z[]
+If you specify multiple ip kernel arguments, the `machineNetwork.cidr` must be the CIDR of the primary network.
+endif::ibm-z[]
 |An array of objects. For example:
```

Comment:
ifdef::ibm-power[]
If you specify multiple ip kernel arguments, the `machineNetwork.cidr` must be the CIDR of the primary network.
endif::ibm-power[]